scripts/prep_data_for_model.R

```diff
@@ -327,7 +327,7 @@ write_csv(rst_with_inclusion_criteria, "data/model-data/daily_catch_unmarked.csv
 weekly_standard_catch_unmarked <- rst_with_inclusion_criteria %>% 
   mutate(week = week(date),
          year = year(date)) %>% 
-  group_by(week, year, stream, site_group, site, subsite, run, lifestage, adipose_clipped, lifestage_for_model, include_in_model) %>% 
+  group_by(week, year, stream, site, subsite, run, lifestage, adipose_clipped, lifestage_for_model, include_in_model) %>% 
```

Comment:
I had to remove `site_group` from this group statement to get it to run. I remember the last time we updated changing this around a bit for Josh and maybe we didn't do it everywhere. Might be good to do a quick pair to make sure this script is all cleaned up (although maybe not super important if we are going to focus on the data repo.

Reply:
Thank you for flagging. We did some back and forth on the `site_group` variable. Lets confirm all changes got integrated but agreed I don't want to spend too much time as our focus will be on the data package.